environment.yml

```diff
@@ -16,6 +16,7 @@ dependencies:
  - shapely>=1.6
  - geopandas>=0.7
  - dask>=2.6.0
+ - toolz>=0.10.0
```

Comment:
I believe this [array] / [complete] stuff is only relevant for pip? Conda installs the equivalent of dask[complete], thus I think `toolz` isn't needed as an explicit dependency of xclim!

Reply:
Understoud! 
```suggestion
```